### PR TITLE
feat(frontend): add tapd config-ui

### PR DESCRIPTION
**Pull request by warren830**
# Summary

add config-ui for tapd(only connections and advanced mode)
tapd.svg picture is not good....

### Does this close any open issues?
relates to #2673

### Screenshots
image
image
image
image
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: config-ui/src/data/Providers.js [hunk 1]
```diff
@@ -59,6 +61,7 @@ const ProviderLabels = {
   AE: 'Analysis Engine (AE)',
   DBT: 'Data Build Tool (DBT)',
   STARROCKS: 'StarRocks',
+  TAPD: 'Tapd',
```
Comment (e2corporation) [inline config-ui/src/data/Providers.js:64]: Should "Tapd" be capitalized or uppercase "TAPD" like JIRA?

File: config-ui/src/data/integrations.jsx [hunk 2]
```diff
@@ -69,6 +70,24 @@ const integrationsData = [
...
+  {
+    id: Providers.TAPD,
+    type: ProviderTypes.INTEGRATION,
+    enabled: true,
+    multiConnection: true,
+    name: ProviderLabels.TAPD,
+    icon: <TapdProvider className='providerIconSvg' width='30' height='30' style={{ float: 'left', marginTop: '5px' }} />,
+    iconDashboard: <TapdProvider className='providerIconSvg' width='40' height='40' />,
+    settings: ({ activeProvider, activeConnection, isSaving, isSavingConnection, setSettings }) => (
+      <JenkinsSettings
```
Comment (e2corporation) [inline config-ui/src/data/integrations.jsx:82]: **TAPD** needs to use it's own `settings` component file, copy `settings/jenkins.jsx` to `settings/tapd.jsx`, update the Component definition and export names, and then import it and change this to `<TapdSettings .....`>.

File: config-ui/src/images/integrations/tapd.svg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (e2corporation) [inline config-ui/src/images/integrations/tapd.svg:1]: In order to make SVGs act properly as responsive React components, we need to make the width/height responsive. Manually edit this svg and set both `width=100` and `height=100`. The correct size scaling will be applied when this svg is called by providerIcons.

Comment (e2corporation) [inline config-ui/src/images/integrations/tapd.svg:1]:
```suggestion
<svg width="100" height="100" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: config-ui/src/data/Providers.js (hunk 1 above)
Comment (warren830) [reply config-ui/src/data/Providers.js]: fixed

File: config-ui/src/data/integrations.jsx (hunk 2 above)
Comment (warren830) [reply config-ui/src/data/integrations.jsx]: fixed

File: config-ui/src/images/integrations/tapd.svg (hunk 3 above)
Comment (warren830) [reply config-ui/src/images/integrations/tapd.svg:1]: Thanks, I just updated the `svg` with our own label

Comment (warren830) [reply config-ui/src/images/integrations/tapd.svg]: fixed

**Review by e2corporation — APPROVED** [review]
@warren830 Nice work, LGTM.

**Comment by e2corporation** [comment]
@warren830 Please confirm Backend changes w/ @klesh.

**Pull request merged**